### DISCUSSION TITLE
如何配置 Google Drive 以同步配置

### DISCUSSION THREAD

**Discussion by palemoky** [immersive-translate/immersive-translate#692] [Help]
### 概述 | Summary

Hi，@theowenyoung 我在同步到云端配置项中，无论点击立即同步还是手动管理，都会弹出授权失败的提示信息，但我找遍所有的配置项，都没有找到有任何授权的入口，请问我需要如何配置才能使用 Google Drive 同步配置？谢谢🙏

![image](https://github.com/immersive-translate/immersive-translate/assets/21208635/43e65dae-b742-49d6-b3e0-516555e27f93)

**Comment by theowenyoung** [comment]
什么浏览器？油猴还是扩展呢

**Reply by palemoky** [reply]
Chrome（114.0.5735.198） 扩展，版本是 V0.7.5
